iothub/service/src/AmqpServiceClient.cs

```diff
@@ -159,12 +159,6 @@ public async override Task SendAsync(string deviceId, Message message, TimeSpan?
             }
         }
 
-        // This call is executed over HTTP.
-        public override Task<PurgeMessageQueueResult> PurgeMessageQueueAsync(string deviceId)
-        {
-            return PurgeMessageQueueAsync(deviceId, CancellationToken.None);
-        }
-
         // This call is executed over HTTP.
```

Comment:
Why // instead of /// doc comment?

Reply:
I don't intend for this to be doc comment but rather a code comment to aid developers during debugging

Reply:
The benefit of doc comments is they'll show up in a tooltip. Not interested in that?

Reply:
In this case, we have this nicely documented in our readme here, so this is mostly for us: https://github.com/Azure/azure-iot-sdk-csharp#iot-hub-service-sdk